src/lib/omerlo/reader/utils/request.ts

```diff
@@ -35,7 +41,7 @@ export async function request<T>(
 
 type DirtyFetchOptions = {
   queryParams?: ApiData;
-  method?: 'get' | 'post' | 'put' | 'delete';
+  method?: string;
```

Comment:
By setting the type to get|post|put|delete, you ensure that you will never receive 'POST;

Reply:
no it doesn't ensure i'll not receive `POST` or `SOMETHING ELSE`. It just return an error from the diagnostique POV. And `get` is a string, it's not a `get` so the diagnostique currenctly return flaky error